File: api-examples/cat/search/js/responseFilter.js
var settings = {
    "async": true,
    "crossDomain": true,
    "url": "https://catalogue.iudx.org.in/catalogue/v1/search?attribute-filter=%28tags,id%29",
    "method": "GET",
  }
  
  $.ajax(settings).done(function (response) {
    console.log(response);
  });

File: api-examples/cat/count/java/responseFilter.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://<catalogue-server-ip>/catalogue/v1/count?attribute-filter=(tags,id)")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();
